rest-api-spec/src/main/resources/rest-api-spec/test/indices.forcemerge/20_wait_for_completion.yml

```diff
@@ -27,6 +29,28 @@
   - match: { task.action: "indices:admin/forcemerge" }
   - match: { task.description: "Force-merge indices [test_index], maxSegments[1], onlyExpungeDeletes[false], flush[true]" }
 
+---
+"Force merge index with wait_for_completion after task description changed":
+  - skip:
+      version: " - 2.12.99 "
+      reason: "task description was changed in 2.13.0"
+      features: allowed_warnings
```

Comment:
Do we need to add `node_selector` features here as well to skip any nodes <= 2.12.99 ?

Reply:
Yeah, we need that, thanks!